app/styles/components/cta-emberconf.css

```diff
@@ -0,0 +1,197 @@
+.emberconf-announcement {
+  align-items: center;
+  background: var(--color-gray-300);
```

Comment:
I didn't pick the next background because I didn't want someone to think that I just oops got it in the wrong place, and the same next color also felt too matchy-matchy instead of coordinated. Make sense?

Reply:
yea makes sense 👍